content/guides/tina-cloud/getting-started/markdown-plugin.md

```diff
@@ -83,7 +83,7 @@ The `ui` property allows you to control the output of the GraphQL-generated form
 
 ### A quick test
 
-If you check the Next Starter now and enter edit mode, you will see the body now has Markdown options to make editing easier:
+Restart your application by shutting it down and running `yarn tina-dev` and enter edit mode, you will see the body now has Markdown options to make editing easier:
```

Comment:
```suggestion
Restart your application by shutting it down and running `yarn tina-dev`. Be sure you're in edit mode and you will see the body now has Markdown options to make editing easier:
```

Just thinking they'll be in edit mode already from the previous steps, but if they're not they no how to get there